File: src/main/java/com/matamercer/microblog/MatablogApplication.java
package com.matamercer.microblog;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.data.redis.RedisRepositoriesAutoConfiguration;

@SpringBootApplication(
		exclude = { RedisRepositoriesAutoConfiguration.class }
)
//@EnableCaching
public class MatablogApplication {
	public static void main(String[] args) {
		SpringApplication.run(MatablogApplication.class, args);
	}

}

File: src/main/java/com/matamercer/microblog/MatablogApplication.kt
package com.matamercer.microblog

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.data.redis.RedisRepositoriesAutoConfiguration
import org.springframework.boot.runApplication

@SpringBootApplication(exclude = [RedisRepositoriesAutoConfiguration::class]) //@EnableCaching
class MatablogApplication

fun main(args: Array<String>) {
    runApplication<MatablogApplication>(*args)
}